truss/templates/control/control/helpers/truss_patch/model_container_patch_applier.py

```diff
@@ -127,6 +142,48 @@ def _apply_system_package_patch(self, system_package_patch: SystemPackagePatch):
         else:
             raise ValueError(f"Unknown python requirement patch action {action}")
 
+    def _apply_config_patch(self, config_patch: ConfigPatch):
+        self._app_logger.debug(f"Applying config patch {config_patch.to_dict()}")
+        TrussConfig.from_dict(config_patch.config).write_to_yaml_file(
+            Path(self._inference_server_home / config_patch.path)
+        )
+
+    def _apply_env_var_patch(self, env_var_patch: EnvVarPatch):
+        self._app_logger.debug(
+            f"Applying environment variable patch {env_var_patch.to_dict()}"
+        )
+        action = env_var_patch.action
+        ((env_var_name, env_var_value),) = env_var_patch.item.items()
+
+        if action == Action.REMOVE:
+            os.environ.pop(env_var_name, None)
+        elif action in [Action.ADD, Action.UPDATE]:
+            os.environ.update({env_var_name: env_var_value})
+        else:
+            raise ValueError(f"Unknown patch action {action}")
+
+    def _apply_external_data_patch(self, external_data_patch: ExternalDataPatch):
+        self._app_logger.debug(
+            f"Applying external data patch {external_data_patch.to_dict()}"
+        )
+        action = external_data_patch.action
+        if action == Action.REMOVE:
+            item = ExternalDataItem.from_dict(external_data_patch.item)
+            filepath = self._data_dir / item.local_data_path
```

Comment:
i don't recall. Is the local_data_path incllude `data/` (relative to truss context root) or is it relative to the data dir?
@pankajroark can you comment here

Reply:
the `local_data_path` is relative to the data dir